gen/typedef.go

```diff
@@ -103,6 +103,26 @@ func typedef(g Generator, spec *compile.TypedefSpec) error {
 				return <equals .Target $lhs $rhs>
 			<- end>
 		}
+
+		<if (eq (zapEncoder .Target) "Object") ->
```

Comment:
The template format supports template-level comments. Let's add a couple to explain what's going on here.

```
</* ... */>
```